tests/integration/test_analysisd/test_eps/conftest.py

```diff
@@ -0,0 +1,159 @@
+import socket
+import shutil
+from typing import List
+import pytest
+
+from wazuh_testing.tools.services import control_service
+from wazuh_testing.tools import configuration, SIMULATE_AGENT, ARCHIVES_LOG_FILE_PATH, \
+                                ALERT_LOGS_PATH, ALERT_FILE_PATH, ALERT_PATH, WAZUH_INTERNAL_OPTIONS
+
+
+@pytest.fixture(scope='function')
+def restart_analysisd_function():
+    """Restart wazuh-analysisd daemon before starting a test, and stop it after finishing"""
+    control_service('restart', daemon='wazuh-analysisd')
+    yield
+    control_service('stop', daemon='wazuh-analysisd')
+
+
+@pytest.fixture(scope='session')
+def configure_local_internal_options_eps(request):
+    """Fixture to configure the local internal options file."""
+    # Define local internal options for vulnerability detector tests
+    local_internal_options = {'wazuh_modules.debug': '2', 'monitord.rotate_log': '0',
+                              'analysisd.state_interval': f"{request.param[0]}"}
+
+    # Backup the old local internal options
+    backup_local_internal_options = configuration.get_wazuh_local_internal_options()
+
+    # Set the new local internal options configuration
+    configuration.set_wazuh_local_internal_options(configuration.create_local_internal_options(local_internal_options))
+
+    yield
+
+    # Backup the old local internal options cofiguration
+    configuration.set_wazuh_local_internal_options(backup_local_internal_options)
+
+
+@pytest.fixture(scope='function')
+def set_wazuh_configuration_eps(configuration, set_wazuh_configuration, configure_local_internal_options_eps):
+    """Set wazuh configuration
+
+    Args:
+        configuration (dict): Configuration template data to write in the ossec.conf.
+        set_wazuh_configuration (fixture): Set the wazuh configuration according to the configuration data.
+        configure_local_internal_options_eps (fixture): Set the local_internal_options.conf file.
+    """
+    yield
+
+
+@pytest.fixture(scope='function')
+def simulate_agent(request):
+    """Fixture to execute the script simulate_agent.py"""
+    # Get IP address of the host
+    hostname = socket.gethostname()
+    IPAddr = socket.gethostbyname(hostname)
+
+    file_excecute = os.path.abspath(SIMULATE_AGENT)
+    subprocess.call(f"python3 {file_excecute} -a {IPAddr} -n {request.param['num_agent']} \
+                    -m {request.param['modules']} -s {request.param['eps']} -t {request.param['time']} \
+                    -f {request.param['msg_size']} -e {request.param['total_msg']} \
+                    -k {request.param['keepalive_disabled']} -d {request.param['receive_msg_disabled']}", shell=True)
+
+    yield
+
+
+def delete_folder_content(folder):
+    """Delete alerts folder content execution"""
+    for filename in os.listdir(folder):
+        filepath = os.path.join(folder, filename)
+        try:
+            shutil.rmtree(filepath)
+        except OSError:
+            os.remove(filepath)
```

Comment:
You should truncate the alert file not remove the alerts folder. This could produce not expected results.
Please use the defined truncate function.

Reply:
If I use the truncate function alerts remain in folders and make tests fail.